features/GRP001_Acyclic_groups.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Acyclic groups and systems
+
+  Scenario: Agreement on IfcGroup (and hence systems) being acyclic
+
+      Then Each IfcGroup must not be referenced by itself directly or indirectly
```

Comment:
Hi, 

Perhaps it's possible to 'standardize' the rule a little bit?

```
@implementer-agreement
@GRP
Feature: GRP001 - Acyclic groups and systems
The rule verifies that all the IfcGroups are acyclic. 

  Scenario: Agreement on IfcGroup (and hence systems) being acyclic

      Then Each IfcGroup must not be referenced by itself directly or indirectly
  
```

Reply:
Thanks, what was I thinking?